src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  padding: 50px 40px;
+  gap: 46px 48px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
And here as well. Remember to use the 'fr' unit when setting the width of the columns in a grid.